ospec/tests/test-ospec.js

```diff
@@ -183,18 +187,6 @@ o.spec("ospec", function() {
 		})
 	})
 
-	o.spec("stack trace cleaner", function() {
```

Comment:
The new cleaner expects that the stack comes from within ospec, with at least one framework function on the stack before we reach user code, breaking this test. I've thus moved it to the new reporter test that provides real ospec stack traces.